orcid-core/src/main/resources/i18n/messages_lr.properties

```diff
@@ -17,2478 +17,2478 @@
 
 #
 #
-#Tue Dec 17 11:07:10 CST 2019
```

Comment:
@leomendoza123 why do you changed so many things here?

Reply:
@amontenegro All this is automatically done by our script `update_test_languages.jy` https://github.com/ORCID/ORCID-Source/blob/ff5a96542d40df534cfdf4f2a176bb7f1e95c6d8/orcid-core/src/main/resources/i18n/update_test_languages.jy

It has been a long time since it's the last run